tests/pkg_emlearn/main.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Emlearn test application
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "model.h"
+
+/* the digit array included must be 4-byte aligned */
+__attribute__((__aligned__(4)))
```

Comment:
Hah, I didn't know this was possible.

Reply:
With the `#include` statement, the preprocessor just replace the line with the content of the file. So in the end, the "generated" code is correct. It's a bit hack :)